syncstorage/storage/__init__.py

```diff
@@ -16,6 +16,9 @@
 
 logger = logging.getLogger("syncstorage.storage")
 
+# Rough guesstimate of the maximum reasonable life span of a batch.
+BATCH_LIFETIME = 60 * 60 * 2  # 2 hours, in seconds
+
```

Comment:
I made this a sharable constant, previously it was hard-coded in several places.